pkg/project/registry/projectrequest/delegated/sample_template.go

```diff
@@ -40,13 +39,13 @@ func DefaultTemplate() *templateapi.Template {
 		oapi.OpenShiftDisplayName:   "${" + ProjectDisplayNameParam + "}",
 		projectapi.ProjectRequester: "${" + ProjectRequesterParam + "}",
 	}
-	if err := templateapi.AddObjectsToTemplate(ret, []runtime.Object{project}, projectapiv1.SchemeGroupVersion); err != nil {
+	if err := templateapi.AddObjectsToTemplate(ret, []runtime.Object{project}, latest.Version); err != nil {
```

Comment:
This is used for a server to loopback to itself.  I don't think I care about making compatible two releases back.

Reply:
Ah, this is against 3.6.

Can you do this in the CLI hook instead of here?

Reply:
This should just be `projectapiv1.LegacySchemeGroupVersion` for 3.6 and earlier.  I have no idea what you mean by CLI hook.

Reply:
> This should just be projectapiv1.LegacySchemeGroupVersion for 3.6 and earlier. I have no idea what you mean by CLI hook.

The CLI has a command dumping this where they want to serialize to old objects.  Where this is used by the apiserver, we don't care.

Reply:
@deads2k I am attempting to convert template objects after making a call to `delegated.DefaultTemplate()` [here](https://github.com/openshift/origin/blob/master/pkg/cmd/server/admin/create_bootstrap_project_template.go#L69), however I am unable to use `kapi.Scheme.ConvertToVersion` as it requires pointers as input. Template objects are originally pointers, but are then [wrapped in a runtime.encodable when they are added to a template](https://github.com/openshift/origin/blob/master/pkg/template/apis/template/helpers.go#L44).

Reply:
> @deads2k I am attempting to convert template objects after making a call to delegated.DefaultTemplate() here, however I am unable to use kapi.Scheme.ConvertToVersion as it requires pointers as input. Template objects are originally pointers, but are then wrapped in a runtime.encodable when they are added to a template.

Ok.  As long as it stays in 3.6 and doesn't move to 3.7, this is ok.  In 3.7 we're using RBAC resources